sdk/template/ci.yml

```diff
@@ -7,7 +7,9 @@ trigger:
     - release/*
   paths:
     include:
-    - sdk/template/
+    - sdk/template/ 
+    # eng/common code changes trigger template pipeline for basic checking.
+    - eng/common/
```

Comment:
Changing this template CI should be unnecessary. sdk/core pipeline should build every project (my understanding is that this is true across language repos), so doing this for any other SDK is redundant and time wasted.

Reply:
Even if we did want to include this for every non-core project, this change would need to be in all track 2 projects' ci.yml. This file is just a template for *new* projects, and turns out isn't the one we want people using anyway (see #27397).